bigbluebutton-html5/imports/ui/components/user-list/user-list-content/waiting-users/component.jsx

```diff
@@ -0,0 +1,73 @@
+import React, { PureComponent } from 'react';
+import PropTypes from 'prop-types';
+import { defineMessages } from 'react-intl';
+import Icon from '/imports/ui/components/icon/component';
+import { Session } from 'meteor/session';
+import { styles } from './styles';
+
+const propTypes = {
+  intl: PropTypes.shape({
+    formatMessage: PropTypes.func.isRequired,
+  }).isRequired,
+};
+
+const intlMessages = defineMessages({
+  waitingUsersTitle: {
+    id: 'app.userList.guest.waitingUsersTitle',
+    description: 'Title for the notes list',
+  },
+  title: {
+    id: 'app.userList.guest.waitingUsers',
+    description: 'Title for the waiting users',
+  },
+});
+
+class WaitingUsers extends PureComponent {
+
+  static toggleWaitingPanel() {
+    Session.set(
+      'openPanel',
+      Session.get('openPanel') === 'waitingUsersPanel'
```

Comment:
Worth adding to https://github.com/bigbluebutton/bigbluebutton/blob/master/bigbluebutton-html5/imports/ui/components/panel-manager/component.jsx ?